x/csr/types/errors.go

```diff
@@ -16,4 +16,5 @@ var (
 	ErrSmartContractSupply = sdkerrors.Register(ModuleName, 1000, "csr::CSRPool")
 	ErrNFTSupply           = sdkerrors.Register(ModuleName, 1001, "csr::CSRPool")
 	ErrMisMatchedNFTSupply = sdkerrors.Register(ModuleName, 1002, "csr::CSRPool")
+	ErrInvalidType = sdkerrors.Register(ModuleName, 6973, "MsgRegisterCSR:: invalid type")
```

Comment:
```suggestion
	ErrInvalidType = sdkerrors.Register(ModuleName, 6973, "csr::MsgUpdateCSR")
```
we want to conform to the same error structure